casepro/msg_board/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django_comments.models import Comment
+
+
+class MessageBoardComment(Comment):
```

Comment:
@rowanseymour it doesn't have to. The `Comment` model is a very [light wrapper of the abstract model](https://github.com/django/django-contrib-comments/blob/master/django_comments/models.py#L12-L14). I don't see any reason not to use it.